doc/integrator/multy_organization.rst.tmpl

```diff
@@ -0,0 +1,102 @@
+. _integrator_multy_organization:
+
+Muty organiration
+=================
+
+Our application can host multiple organizations with small changes between organization.
+
+In a multi organizations each organization will have the came code (Python, Javascript, SCSS).
+
+In this example we will have the came CSS but we can do some variation by using CSS var,
+see `cssVars` in `gmfOptions` in the ngeo GMF constants definitions
+`gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+The following lines will profide a relay basic implementation for muty organiration
+
+__init__.py
+-----------
+
+In the file `geoportal/<package>_geoportal/__init__.py` add the folloring lines:
+
+.. code:: python
+
+
+def get_instance_prefix(request):
+    if request.host == 'org1.camptocamp.com':
```

Comment:
Is it clear to the reader that these are example host names and that the code will need to be adapted to the real host name?

Reply:
I just add the following introduction:

The code should be adapted, actually he handles the hostname 'org1.camptocamp.com' and 
'org2.camptocamp.com', and you probably want to put the hardcoded values in the config.